build.gradle

```diff
@@ -47,6 +47,9 @@ subprojects {
         withJavadocJar()
         withSourcesJar()
     }
+    tasks.withType(Jar) {
+        destinationDirectory = file("$rootDir/build-artifacts")
```

Comment:
This is the gradle config to send JARs to `build-artifacts`. `./gradlew clean` doesn't know about the location change, though, so `make clean` takes care of them.